LocalSettings.php

```diff
@@ -2378,6 +2378,15 @@
 	'wgMathValidModes' => [
 		'default' => [ 'mathml' ],
 	],
+	
+	// New User Email Notification
+	
+	$wgNewUserNotifEmailTargets => array[
```

Comment:
Needs to be

	'wgNewUserNotifEmailTargets' => [
 		'default' => [],
 		'default' => [ 'gustav@nyvell.net' ],
 	],

Reply:
err second default should be femmanwiki